charts/alfresco-repository/templates/_helpers-search.tpl

```diff
@@ -15,19 +15,14 @@ Usage: include "alfresco-repository.search.flavor.valid" "FLAVOR"
 {{/*
 Check whether a Solr shared secret was provided
```

Comment:
check because it doesn't seems to reflect the current implementation

Reply:
yes it is... the description talks about shared secret as in general authentication concept, not aé kubernetes secrets shared by different charts